Makefile

```diff
@@ -1,5 +1,2 @@
 test:
-	rm -rf /tmp/_adareducer_test
-	cp -R tests/orig /tmp/_adareducer_test && \
-    gprbuild -P /tmp/_adareducer_test/p.gpr && \
-    python adareducer.py --single-file /tmp/_adareducer_test/proc.adb --follow-closure /tmp/_adareducer_test/p.gpr tests/basic.sh
+	bash tests/test.sh
```

Comment:
This hunk and the new file belong to another commit. Also, can you restore the linefeed character at the end of the file?

Reply:
Done and done!

Reply:
Thanks!